datumaro/tests/test_cvat_format.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+import os
+import os.path as osp
+from xml.etree import ElementTree as ET
+
+from unittest import TestCase
+
+from datumaro.components.extractor import (Extractor, DatasetItem,
+    AnnotationType, PointsObject, PolygonObject, PolyLineObject, BboxObject,
+    LabelCategories,
+)
+from datumaro.components.importers.cvat import CvatImporter
+import datumaro.components.formats.cvat as Cvat
+from datumaro.util.image import save_image
+from datumaro.util.test_utils import TestDir
+
+
+class CvatExtractorTest(TestCase):
+    @staticmethod
+    def generate_dummy_cvat(path):
+        images_dir = osp.join(path, Cvat.CvatPath.IMAGES_DIR)
+        anno_dir = osp.join(path, Cvat.CvatPath.ANNOTATIONS_DIR)
+
+        os.makedirs(images_dir)
+        os.makedirs(anno_dir)
+
+        root_elem = ET.Element('annotations')
```

Comment:
@zhiltsov-max , I will vote for a readable form and parse xml from a string. Why does it necessary to construct cvat xml this way?